modules/experimental/pages/portal.adoc

```diff
@@ -0,0 +1,139 @@
+= Portal
+
+`Unit ability`
+footnote:[
+include::special-rules:partial$no-such-ability-in-e40k.adoc[]
+]
+
+Some races employ technologies that allow them to open trans-dimensional portals on the battlefield.
+This can be an effective method to deploy reserve forces at the optimum time and place.
+
+Well-known examples include Necron portals and Eldar Webway portals.
+
+---
+
+== Data sheet variations of {portal}
+
+Where we include {portal} on a unit data sheet -- or for an objective -- we also include some further information, as attributes in brackets.
+For example, like this: `{portal} _(INF, 4+, {walker}, Auto)_`.
+
+Mostly, these attributes indicate what other friendly units can actually pass through the portal -- that is, which kinds of unit that portal permits you to embark/disembark during the game.
+
+* `_(INF)_`: Permits Infantry units.
+* `_(4+)_`: Permits Vehicle units that have an Armour value of `4+` or lower.
+* `_({walker})_`: Permits Vehicle units that have the {walker} ability, regardless of their Armour value.
+
+We also indicate how the portal affects the arrival of detachments that you intend to use with it, through the `_(Roll)_` or `_(Auto)_` attributes.
+See <<Portal Reserves>> for more about this.
+
+TIP: We use `_(Roll)_` and `_(Auto)_` to represent different technologies for different factions.
+They are mutually exclusive, for both thematic and practical reasons -- so {portal} units in a given army list must all be of one of these types, never a mix of both.
+
+== Game effects of {portal}
+
+Where your army includes any units or objectives that have the {portal} ability:
+
+* Regardless of the scenario, you may keep any detachments that contain units that can use a portal out of the play area, as <<Portal Reserves>>.
+
+Where your unit or objective has the {portal} ability:
+
+* You can use this unit to move portal reserves detachments into play after they arrive.
+You can also move detachments that are in play between one portal and another.
+See <<portal-moves>>.
+
+[[portal-moves]]
+== Use a {portal} unit to move your forces
+
+You can use portals to move some of your forces around.
+Where you wish to move one of your detachments into or out of a portal, and it contains permitted units:
+
+* You may do this only in the Movement phase.
+// TODO: Is this stipulation necessary? On the one hand we're trying to apply standard transport rules for simplicity, but on the other hand this kind of stipulation sets up differences from those rules which complicates things again and creates logical gaps.
```

Comment:
Yes, it's 100% necessary! 

If detachments can move through Portals in the Assault phase then they can be in a position of complete safety during the Shooting phase and then charge through the Portal, with no opportunity to shoot them first. This breaks a fundamental part of the game, where you get to shoot assaulters before they assault, unless the assaulters are exceptionally fast or have outmanoeuvred you enough that they can hide behind terrain.

In the same way, a breaking detachment could retreat through a Portal to a safe part of the table, even if they were completely surrounded by enemy units.

Reply:
Okay, that makes sense to me.

The difficulty I'm having is that on the one hand you're saying that this is all just standard transportation rules, but on the other hand as I edit I find little logical problems that seem to imply caveats or exceptions that aren't explicit.

In this case, given what you've just explained, we effectively seem to have "just like the standard transport rules... except that you can't use it in the Assault phase". Such an exception is fine in principle, but I have to find, dissect and understand it so that I can edit effectively and make it more explicit and unambiguous in the text.

We might ultimately need to add a statement somewhere to the effect that "this works just like the standard transportation rules, with these exceptions and clarifications: [...]"

Reply:
'You may do this only in the Movement phase.' is explicit isn't it?

Reply:
> 'You may do this only in the Movement phase.' is explicit isn't it?

In itself, yes — but the point that I'm trying to make is that at present it demands close attention to figure out the extent to which embarking/disembarking a portal does or does not actually play out "just like _Transport (X)_". If I'm finding it tricky as a technical writer then it's likely to be tricky for others.

Given that they're **close** but not quite **exactly** the same in practice, I believe that we'll need to make it easier to compare and contrast so that the reader can grasp the differences quickly. How we can do that should become clear by the time we've finished this round of editing with comment threads like this one to clarify specific points.

Reply:
For now at least, I'll insert a note near the top of the topic to mention the fact that these things are similar but not quite the same. At least that will give players fair notice.

So, at the same time I'll remove this TODO note and resolve this thread.
```suggestion
```